minio/mcs
# [WIP] Added bulk functionality for add users to groups

## What does this do?

Added functionality in users module to add multiple users to multiple groups at once.

## How to test

1. Go into Users module and check that the add to group is disabled when no users are selected in the list
2. Select multiple users in the list, the add to group button should get enabled
3.  Click on the add to group button, a modal box to select multiple groups should be opened
4. Select the desired groups and click on the save button, the modal box should close and the users should get unchecked
5. Check on each user & review the groups, the user must have assigned the selected groups on step 4
